app/src/main/java/org/stepic/droid/ui/custom/LanguageTextView.kt

```diff
@@ -0,0 +1,103 @@
+package org.stepic.droid.ui.custom
+
+import android.content.Context
+import android.os.Parcel
+import android.os.Parcelable
+import android.support.annotation.ColorRes
+import android.support.annotation.DrawableRes
+import android.support.v7.widget.AppCompatTextView
+import android.util.AttributeSet
+import android.view.View
+import android.widget.Checkable
+import org.stepic.droid.R
+import org.stepic.droid.util.ColorUtil
+
+class LanguageTextView
```

Comment:
Can the same behavior be achieved with styled radio buttons?

Reply:
May be, I didn't research enough. We can try it in future releases, when number of languages will grow.